services/post_service.py

```diff
@@ -0,0 +1,10 @@
+from api_schemas.post_schemas import PostCreate
+from sqlalchemy.orm import Session
+from db_models.post_model import Post_DB
+
+
+def create_new_post(data: PostCreate, db: Session):
+    post = Post_DB(name=data.name, council_id=data.council_id)
```

Comment:
Here it would be nice take the council id and look up that council from DB. raise a 404 if not found before creating the post. otherwise it's 500 Internal Server Error if someone supplies a bad id